storymap_templates/gdh_default_storymap_template.yaml

```diff
@@ -69,134 +81,195 @@ Acknowledgements_Content: &Acknowledgements |
-  - type: text
-    style: paragraph
-    text: *Geosurvey
-
-  - type: image
-    url: "https://community-6fcc.kxcdn.com/uploads/default/original/1X/0eed20c97d74c71bc9f937b5bea3a8b2f8412a6b.jpeg"
-    caption: "Graph"
-
-  - type: text
-    style: paragraph
-    text: *Geosurvey2
-
-  - type: table
-    headers:
-      - "Improvement Suggestion"
-      - "Comment"
-    rows:
-      - ["Better infrastructure", "Improved road networks"]
-      - ["Increase greenery", "Plant more trees in urban areas"]
-      - ["Flood prevention", "Build retention ponds"]
-
-  - type: separator
-
-  # Workshop_Schedule
-  - type: text
-    style: heading
-    text: "6. Workshop Schedule"
-
-  - type: text
-    style: paragraph
-    text: *Workshop_Schedule
-
-  - type: image
-    url: "https://community-6fcc.kxcdn.com/uploads/default/original/1X/ef0664e002d38cbbad258bbcdfdfcdc7cf985a6e.jpg"
-    caption: "Original Source: Emil Lanfranchi and Francesco Fonzino"
-
-  - type: separator
-
-  # Participants_of_the_workshop
-  - type: text
-    style: heading
-    text: "7. Participants of the workshop"
-
-  - type: text
-    style: paragraph
-    text: *Participants_of_the_workshop
-
-  - type: gallery
-    images:
-      - url: "https://community-6fcc.kxcdn.com/uploads/default/original/1X/b9e8c9bd2e9fc7eaf6c3280335ff158cc68be7c7.jpg"
-        caption: "Photo 1"
-      - url: "https://community-6fcc.kxcdn.com/uploads/default/original/1X/27979335c966086ea51b2e76d0a6cc799ea7ba81.jpg"
-        caption: "Photo 2"
-      - url: "https://community-6fcc.kxcdn.com/uploads/default/original/1X/d38d45ad6bb1af098903efae70de63da018d7071.jpeg"
-        caption: "Photo 3"
-      - url: "https://community-6fcc.kxcdn.com/uploads/default/original/1X/7af77de5e8efb689e8e94ca32c33c3bfffc3e275.jpg"
-        caption: "Photo 4"
-    caption: "Photos"
-    
-  - type: text
-    style: heading
-    text: "Acknowledgements"
-
-  - type: text
-    style: paragraph
-    text: *Acknowledgements
-
-  - type: image
-    url: "https://igcollab-com.maps.arcgis.com/sharing/rest/content/items/3bc45b161a46412d86cd7c4daabfe7f8/data"
-    caption: "Partner Logos"
-
-  - type: image
-    url: "https://www.rug.nl/library/open-access/blog/which-creative-commons-licence-should-i-choose-19-02-2021!attachment?c=images/blog-image-normal.jpg"
-    caption: "Except where noted, images are licensed under Creative Commons License."
\ No newline at end of file
+sections:
+  - title: "Introduction"
+    panels:
+      - type: text
+        style: subheading
+        text: "1. Background"
+      - type: text
+        style: paragraph
+        text: *Background
+      - type: separator
+      - type: text
+        style: subheading
+        text: "2. Area of Interest"
+      - type: map
+        item_id: "{negotiated_design_item_id}"
+        caption: "Figure 1: Design Area"
+      - type: text
+        style: paragraph
+        text: *Area_of_Interest
+
+  - title: "Negotiation Details"
+    panels:
+      - type: text
```

Comment:
While thinking about this, perhaps `content_type` is more explicit than just `type`, not sure how much it is to change at this time.

Reply:
hank you for the suggestion! I’ve updated the code to use content_type instead of type. Please review the changes.